src/main/java/edu/wpi/grip/core/SocketHint.java

```diff
@@ -69,7 +70,9 @@ public View getView() {
     }
 
     public T getDefaultValue() {
-        return defaultValue;
+        Cloner cloner = new Cloner();
```

Comment:
Are you sure this works with OpenCV `Mat`s?  Maybe we just want to call the object's `.copy()` method.